src/style.css

```diff
@@ -1 +1,69 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  width: 100%;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-left: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  align-items: center;
+
+  padding: 0 40px;
```

Comment:
Why don't you put margin and padding together?

Reply:
To have that 50px padding right , i had to put 40px so it can match , i know is not the right way , my fault is that after i matched the design i didn't review my styles. i know it's a mess